docs/index.html

```diff
@@ -20,21 +20,22 @@ <h1>image edit tools</h1>
             <li><a href="#trim" data-toggle="tab">trim</a></li>
             <li><a href="#reverse" data-toggle="tab">reverse</a></li>
             <li><a href="#filter" data-toggle="tab">filter</a></li>
+            <li><a href="#tile" data-toggle="tab">tile</a></li>
             <li><a href="#extension" data-toggle="tab">extension</a></li>
         </ul>
         <div class="tab-contents">
             <div class="tab-content" id="resize">
                 <label>width:<input type="number" max="5000" min="0" step="1" id="resize-width">px</label>
                 <label>height:<input type="number" max="5000" min="0" step="1" id="resize-height">px</label>
                 <label>ratio:<input type="number" max="10" min="0.01" step="0.01" id="resize-ratio"></label>
-                <button class="button" onClick="resize();">execute</button>
+                <button class="button need-ckeck" onClick="resize();">execute</button>
             </div>
             <div class="tab-content" id="trim">
                 <label>left:<input type="number" max="5000" min="0" step="1" id="left">px</label>
                 <label>top:<input type="number" max="5000" min="0" step="1" id="top">px</label>
                 <label>width:<input type="number" max="5000" min="0" step="1" id="trim-width">px</label>
                 <label>height:<input type="number" max="5000" min="0" step="1" id="trim-height">px</label>
-                <button class="button" onClick="trim();">execute</button>
+                <button class="button need-check" onClick="trim();">execute</button>
```

Comment:
I couldn't find any reference to need-check, can we remove it as an unnecessary class?

Reply:
Remove it and there is no problem.
I was actually planning to use this class to do validation on the javascript side when the button is pressed.
But even if it was INVALID, I could not abort the process on the go side.
So I deleted the javascript side processing, but forgot to delete that class :sweat:

Reply:
I thought it was somewhat like that.
Don't worry.
Can you have it removed?

Reply:
Thank you for your precious opinion.
I have decided on my thoughts and will tell you.

We should create a function on the go side that sets the appropriate max value on the browser side, and call it from the browser side at the appropriate timing.
Since the go side knows the appropriate max value, it can be validated at runtime.

Since the above is not related to the implementation that this PR should do, this PR does not need to be modified in any particular way.